src/spec-lang/ast/for_all.ts

```diff
@@ -54,19 +53,21 @@ export class SForAll extends SNode {
                 start === undefined && end === undefined,
                 `Can't specify both an array and start or end for forall.`
             );
-            this.start = new SNumber(bigInt(0), 10);
-            this.end = new SMemberAccess(array, "length");
+            // this.start = new SNumber(bigInt(0), 10);
+            // this.end = new SMemberAccess(container, "length");
```

Comment:
Should we completely remove these commented lines?

Reply:
Good catch. Fixed in a97c7d9.